openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -9,18 +10,45 @@ class IntegrateKitsuNote(pyblish.api.ContextPlugin):
     order = pyblish.api.IntegratorOrder
     label = "Kitsu Note and Status"
     families = ["render", "kitsu"]
+
+    # status settings
     set_status_note = False
     note_status_shortname = "wfa"
 
+    # comment settings
+    custom_comment_template = {
+        "enabled": False,
+        "comment_template": "{comment}",
+    }
+
+    def safe_format(self, msg, **kwargs):
+        """If key is not found in kwargs, set None instead"""
+
+        def replace_missing(match):
+            key = match.group(1)
+            if key not in kwargs:
+                self.log.warning(
+                    "Key `{}` was not found in instance.data "
+                    "and will be rendered as `` in the comment".format(key)
+                )
+                return ""
+            else:
+                return str(kwargs[key])
+
+        pattern = r"\{([^}]*)\}"
+        return re.sub(pattern, replace_missing, msg)
+
     def process(self, context):
         # Get comment text body
         publish_comment = context.data.get("comment")
         if not publish_comment:
             self.log.info("Comment is not set.")
 
-        self.log.debug("Comment is `{}`".format(publish_comment))
-
         for instance in context:
+            # Check if instance is a review by checking its family
+            if "review" not in instance.data["family"]:
```

Comment:
The instance might have the family `kitsu` but isn't a review. In that instance it would still be processed in the plugin.

Reply:
```suggestion
            if "review" not in instance.data["families"]:
```
If @Tilix4's suggestion is wrong then likely it should still have to be this if we need to filter. :)

But note that the context has all instances, so also the workfile instance which is likely not marked as `review` but we might still want to integrate a note to kitsu about it? Or are notes ONLY for reviewables?

Reply:
I found what @Tilix4 wrote on Discord very logical:

> in our current Blender refactor, we basically dropped it because of the inconsistent behaviour you describe. IMO, you publish a review only if you have a review instance in your scene, otherwise it is too hard to track what's going on.

Else the end user would see something like this (is not using this PR to get extra information per comment):
![image](https://user-images.githubusercontent.com/49758407/225572606-bfb0a64b-c940-4311-bcff-ad5655b06b58.png)

So I would say only reviews would be uploaded to Kitsu

Reply:
I agree with @EmberLightVFX who agrees with me.